package-tracking/settings.py

```diff
@@ -25,7 +25,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ['10.0.0.9']

```

Comment:
Need to do that for layout testing on mobile

Reply:
But it doesn't belong in the commit, since its not production, but located in production-code area. Remove it.

Reply:
I mean it's not really a push to production... It could be changed on the push between dev and main but whatever, you win

Reply:
By production I meant 2 things:
- In `dev` branch. So it's likely it will be missed and could end up in `master`.
- In _production-area_ code. You can put _dev-only_ code, but make sure it's separated explicitly.